webapp/templates/jury/contest.html.twig

```diff
@@ -356,7 +356,13 @@
                             <td>
                                 <a href="{{ link }}">
                                     {% if problem.lazyEvalResults is not null %}
-                                        {{ problem.lazyEvalResults | printYesNo }}
+                                        {% if problem.lazyEvalResults == 1 %}
```

Comment:
@nickygerritsen how can I use the constants here also?

Reply:
`constant(‘FQDN'`) which is horrible